src/screens/CommunityHome/index.tsx

```diff
@@ -256,7 +261,10 @@ export default function CommunityHome({ route }: any) {
             source={
               communityData?.data.avatarFileId
                 ? {
-                    uri: `https://api.${apiRegion}.amity.co/api/v3/files/${communityData?.data.avatarFileId}/download?size=medium`,
+                    uri: getAvatarURL(
```

Comment:
i think, region is assigned in sdk already

Reply:
e.g. https://github.com/EkoCommunications/AmityUiKitWeb/blob/develop/src/core/hooks/useImage.ts

Reply:
Thanks. updated krub :)